examples/pytorch/text-classification/run_glue_no_trainer.py

```diff
@@ -188,6 +196,18 @@ def main():
     if args.seed is not None:
         set_seed(args.seed)
 
+    # Handle the repository creation
+    if accelerator.is_main_process:
+        if args.push_to_hub:
+            if args.hub_model_id is None:
+                repo_name = get_full_repo_name(Path(args.output_dir).name, token=args.hub_token)
+            else:
+                repo_name = args.hub_model_id
+            repo = Repository(args.output_dir, clone_from=repo_name)
+        elif args.output_dir is not None:
```

Comment:
Can this arg be None? Wouldn't this lead to a failure later?

Reply:
Also we assume in the if above that `args.output_dir` is not None already no in:

```python
repo_name = get_full_repo_name(Path(args.output_dir).name, token=args.hub_token)
```

=> maybe raise Error before the repo creation if `args.output_dir` is None?

Reply:
This is asserted [here](https://github.com/huggingface/transformers/blob/f94cb9e91bb8a74a583c41396e3a5a22e6360e87/examples/pytorch/text-classification/run_glue_no_trainer.py#L167) already :-)